catalogs/auc.yaml

```diff
@@ -7,8 +7,8 @@ sources:
     description: "Al-Musawwar Magazine Collection"
     driver: iiif_json
     metadata:
-      data_path: auc/iiif/al_musawwar
-      config: auc_iiif_config
+      data_path: auc/al_musawwar
+      config: auc
```

Comment:
I think these configs should be `auc_iiif`

Reply:
we will only have one config for auc so no need for the iiif ending

Reply:
Will you be fixing that in a follow up PR or in the existing transform PR? This needs to match the path of the config used as it's pulled in the report.